tools/scripts/deploy.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" && pwd)"
+rootdir="${scriptdir%/*/*}"
+
+dbg() {
+    [ "$VERBOSE" = "true" ] && echo "$@"
+}
+
+err() {
+    printf "\033[1;31m%s\n\033[0m" "$@"
+}
+
+success() {
+    printf "\033[1;32m%s\n\033[0m" "$@"
+}
+
+run() {
+    dbg "$*"
+    "$@" || exit $?
+}
+
+
+usage() {
+    echo "usage: $(basename "$0") [-hvdb] [--target <user@target>] [--proxy <user@proxy>]"
+}
+
+main() {
+
+    if ! OPTS=$(getopt -o 'hvdb:' --long help,verbose,deploy,build-dir:,target:,proxy: -n 'parse-options' -- "$@"); then
+        err "Failed parsing options." >&2
+        usage
+        exit 1 
+    fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -v | --verbose)       VERBOSE=true; shift ;;
+            -h | --help)          usage; exit 0 ;;
+            -d | --deploy)        DEPLOY=true; shift ;;
+            -b | --build-dir)     BUILD_DIR="$2"; shift; shift ;;
+            --proxy)              SSH_OPTIONS="$SSH_OPTIONS \"-oProxyJump $2\""; shift 2 ;;
```

Comment:
I believe the commit message says this will be added later, but in fact it has already been added here, right?